### ISSUE TITLE
Some icons don't show properly.

### ISSUE THREAD

**Issue by neoOpus** [fwextensions/QuicKey#99]
Github for some reason doesn't appear and internal browser icons also don't show

![image](https://user-images.githubusercontent.com/1388336/225235265-670c03f3-9971-47e3-9cde-02b0e20d481c.png)

**Comment by fwextensions** [comment]
Looks like you're using Edge?  QuicKey will show whatever the browser returns for a `chrome://favicon/` URL, like `chrome://favicon/edge://gpu/`.  I'd have to do special handling for those tabs, which doesn't seem worth it.

The GitHub favicons should show up, though.  They're a little hard to see in dark mode, but this is what I get in Edge:

![image](https://user-images.githubusercontent.com/61631/225829667-eddc8285-694e-4439-82e8-3c4a25155ea3.png)

Are you using some other extension that changes the GitHub favicon?

**Comment by neoOpus** [comment]
Hi,

Actually Github icons on Edge Dark theme shows as white

![image](https://user-images.githubusercontent.com/1388336/226623730-f53aa6a6-41be-483d-a2ad-240b3c4203ca.png)

I will look for a solution or come with some suggestions.

**Comment by fwextensions** [comment]
QuicKey will show whatever the tab's current favicon is, as reported by the browser.  Other extensions can override that, so that's how they'll show up in QK as well.  

Also, when the browser theme changes, the favicon might not update until that tab gets focused again.  Here, Chrome is in dark mode, and the first GitHub tab was suspended by another extension, and the bottom one is live.

![image](https://user-images.githubusercontent.com/61631/227638796-4b3afe62-c046-41ed-8d60-10025b9e2817.png)

Then I switched the OS to light more, and the lower GitHub icon was still showing the dark mode version:

![image](https://user-images.githubusercontent.com/61631/227639032-37780764-ea3b-4280-8d2d-8afac9db600f.png)

After I switched back to that tab, it updated.
